v3/cypress/support/elements/toolbar-elements.ts

```diff
@@ -0,0 +1,41 @@
+export const ToolbarElements = {
+  getToolshelfSelector(component) {
+    let el = ""
+    switch (component) {
+      case "graph":
+        el = "[data-testid=tool-shelf-button-Graph]"
+        break
+      case "slider":
+        el = "[data-testid=tool-shelf-button-Slider]"
+        break
+      case "calculator":
+        el = "[data-testid=tool-shelf-button-Calc]"
+        break
+      case "table":
+        el = "[data-testid=tool-shelf-button-table]"
+        break
+      case "map":
+        el = "[data-testid=tool-shelf-button-Map]"
```

Comment:
Not super important, but should we make these IDs consistent? All of them capitalize the component name part except`tool-shelf-button-table`. If it were up to me, I'd make all of them all lowercase, but maybe there's a good reason some use uppercase for the component name?